README.md

```diff
@@ -12,125 +13,145 @@ npm install @girder/components
 yarn add @girder/components
 ```
 
-### Use as a prebuilt UMD module
+[VueCLI](https://cli.vuejs.org/) is required for building applications with Girder web components.
+However, a few additional packages must still be manually installed:
+```bash
+npm install -D sass-loader node-sass pug-plain-loader pug
+# or
+yarn add -D sass-loader node-sass pug-plain-loader pug
```

Comment:
I wish there were a better way to communicate these requirements within the framework of package.json.

I still sorta think we could (ab)use `peerDependencies` for these since girder-web-components will never have a use for the traditional `peerDependency` use case.

Reply:
Probably not the right place for this discussion.  I just think this info is better communicated through package.json than README.md

Reply:
I don't think so. This is abusing the fact that `peerDependencies` print warnings and not cause the build to fail when not satisfied, but `peerDependencies` are not mere informational suggestions, they have behavior which relates to how conflicting dependencies are resolved.

The long-term solution / "right answer" is for GWC to publish ES5-compatible module code, so downstreams are not burdened with having to apply a Pug / Sass build process just to import GWC.

Reply:
> The long-term solution / "right answer" is for GWC to publish ES5-compatible module code

That's what the UMD module is for, are you also suggesting we should publish source files transpiled to ES5? Are there good tools to make that easy?

Reply:
You can't do tree-shaking downstream from a monolithic UMD file, and you also don't want to push any potentially-hard-to-build features (anything from Pug usage to basic ES6 features which would need to go though Babel) downstream either. So yes, the solution is to publish ES5-compatible modules on a per-file basis.

Here's some reading:
* https://github.com/rollup/rollup/wiki/pkg.module
* https://webpack.js.org/guides/author-libraries/#final-steps
* https://medium.com/webpack/webpack-and-rollup-the-same-but-different-a41ad427058c